loadtest/user/userentity/scheduled_posts.go

```diff
@@ -0,0 +1,76 @@
+package userentity
+
+import (
+	"context"
+	"github.com/mattermost/mattermost/server/public/model"
+)
+
+func (ue *UserEntity) CreateScheduledPost(teamId string, scheduledPost *model.ScheduledPost) error {
+	user, err := ue.getUserFromStore()
+	if err != nil {
+		return err
+	}
+
+	scheduledPost.UserId = user.Id
+	createdScheduledPost, _, err := ue.client.CreateScheduledPost(context.Background(), scheduledPost)
+	if err != nil {
+		return err
+	}
+
+	err = ue.store.SetScheduledPost(teamId, createdScheduledPost)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (ue *UserEntity) UpdateScheduledPost(teamId string, scheduledPost *model.ScheduledPost) error {
+	user, err := ue.getUserFromStore()
+	if err != nil {
+		return err
+	}
+
+	scheduledPost.UserId = user.Id
+	updatedScheduledPost, _, err := ue.client.UpdateScheduledPost(context.Background(), scheduledPost)
+	if err != nil {
+		return err
+	}
+
+	err = ue.store.SetScheduledPost(teamId, updatedScheduledPost)
+	if err != nil {
+		return err
+	}
+
+	ue.Store().UpdateScheduledPost(teamId, updatedScheduledPost)
+
+	return nil
+}
+
+func (ue *UserEntity) DeleteScheduledPost(scheduledPostId string) error {
+	_, _, err := ue.client.DeleteScheduledPost(context.Background(), scheduledPostId)
+	if err != nil {
+		return err
+	}
+
+	ue.Store().DeleteScheduledPost(scheduledPostId)
+	return nil
```

Comment:
Right now DeleteScheduledPost doesn't return an error, but maybe it should (I left a comment for that). If we decide that's a good decision, then we need to return its error.

Reply:
I don't think its of any use returning error from delete method, it doesn't matter